tsl/test/expected/continuous_aggs_ddl-10.out

```diff
@@ -829,7 +833,7 @@ SELECT * FROM drop_chunks_view ORDER BY time_bucket DESC;
           10 |  14
            5 |   9
            0 |   4
-(8 rows)
+(10 rows)
```

Comment:
Same question here w.r.t. change in output.

Reply:
that cagg didnt get its parameters changed and was already using the new behaviour